src/main.ts

```diff
@@ -95,21 +96,28 @@ async function bootstrap() {
   const expressSessionSecret = configService.get<string>(
     "expressSessionSecret",
   );
+  const mongoUrl = configService.get<string>("mongodbUri");
+
   if (expressSessionSecret) {
     app.use(
       session({
         secret: expressSessionSecret,
         resave: false,
         saveUninitialized: true,
+        store: MongoStore.create({
```

Comment:
shall we leave space for choosing if to use mongoStore or in memory one?

Reply:
+1
I would add a property in configuration to select the current behavior or session in mongodb.